src/main/java/com/github/manolo8/darkbot/utils/StartupParams.java

```diff
@@ -41,6 +41,13 @@
     private static final String NO_OP_COMMAND = COMMAND_PREFIX + "no-op";
     private boolean forceNoOp = false;
 
+    /**
+     * Command-line argument for set start-config, has no parameters
+     * Used for debugging
+     */
+    private static final String START_CONFIGS = COMMAND_PREFIX + "configs";
+    private String startConfigs = null;
```

Comment:
```suggestion
    /**
     * Command-line argument for set start-config, has no parameters
     * Used for debugging
     */
    private static final String START_CONFIG = COMMAND_PREFIX + "config";
    private String startConfig = null;
```
Config, it's a single one, not plural. Also fix the copy-pasted comment